### ISSUE TITLE
Custom markdown for OTP email cannot be deployed

### ISSUE THREAD

**Issue by EeeasyBreezy** [auth0/auth0-deploy-cli#425]
### Description

Current implementation of auth0-deploy-cli does not import OTP email markdown for email connection
Instead, a path to file, which contains this markdown, is interpreted as markdown itself and imported into auth0 tenant

Expected: html markdown from corresponding file must be imported

### Reproduction

1. Export configuration in yaml format from Auth0 tenant
2. Set up email connection configuration  in yaml file with section below:
```
- name: email
      strategy: email
      is_domain_connection: false
      options:
          name: email
          totp:
              length: 6
              time_step: 180
          email:
              body: ./emails/otp.html
              from: <no-reply@xxx.com>
              syntax: liquid
              subject: Welcome to xxx
          disable_signup: true
          brute_force_protection: true
```
3. Make sure file `./emails/otp.html` exists and contains some valid html markup 
4. Invoke import operation using `auth0-deploy-cli` util
5. Go to auth0 management panel, open email connection and observe that instead of html mark up from `./emails/otp.html` `Body` field contains path to file from tenant config

### Environment

auth0-deploy-cli@7.3.7 is used

**Comment by willvedd** [comment]
@EeeasyBreezy Thanks for originally opening this issue. It's unclear to me if you've replaced the template with reference to a file. From the reproduction instructions, it seems like you are creating that configuration by scratch and expecting the body to be read from your path. The software is currently unable to discern paths from regular strings, so creating a path will not work as you may expect.

I'm opting to close this ticket for now as it is not actionable for us, however I'd be happy to reopen if you're able to provide more context.

**Comment by EeeasyBreezy** [comment]
> @EeeasyBreezy Is the path to that HTML file generated on export? Or did you manually create that path yourself?

Sorry for huge delay in response
Had some busy days recently :) 

Answer to the question is yes and no at the same time.
When I did export using auth0 util to files on my local machine I obtained another path in YAML file.
After that I changed the path to make config compatible with my current file structure.

I also tried the following:
- Export configuration from auth0 to local files
- Import configurations without changes back to tenant
- Make some changes to OTP email HTML
- Passwordless email HTML still gets overridden by path of the file, not its content

I also have other parts of my Auth0 configuration where I use custom HTML for emails (for example, user verification email).
During export from Auth0 to local files I got path to file in YAML + file with HTML content.
Then when I make changes to this HTML and do import back to Auth0 I can observe these changes in my HTML in Auth0 management panel

Does this information help to clear the issue?
